eos/observable.cc

```diff
@@ -292,8 +304,34 @@ namespace eos
     }
 
     void
-    Observables::insert(const QualifiedName & name, const ObservableEntry * entry) const
+    Observables::insert(const QualifiedName & name, const std::string & latex, const Options & forced_options, const std::string & input_expression) const
```

Comment:
Hm, the naming is a bit weird. ``input_expression`` is not an expression yet. How about ``input`` and ``expression`` in lieu of ``input_expression`` and ``parsed_expression``? Make the code a bit more readable I think.